src/theme/components/slider.ts

```diff
@@ -8,13 +8,15 @@ const baseStyle = (props: Record<string, any>) => {
       `${simplifiedColorScheme}.200`
     )(props),
     trackColor: mode('gray.200', 'gray.700')(props),
+    sliderSizeOnWeb: 200,
```

Comment:
If you adding something in theme, please add some standard style prop. Ideally should rethink it and calculate it based on size prop only (inside the component itself)